recipes/symengine/bld.bat

```diff
@@ -0,0 +1,19 @@
+mkdir build
+cd build
+
+cmake ^
+    -G "%CMAKE_GENERATOR%" ^
```

Comment:
It's all about the quotes. Earlier `CMAKE_GENERATOR` was `Visual Studio 14 2015` (without quotes). So when using in cmake, quotes were added `"%CMAKE_GENERATOR%"`.

Now, `CMAKE_GENERATOR` is `"Visual Studio 14 2015"` (with quotes). So, previous recipes with `"%CMAKE_GENERATOR%"` would become `""Visual Studio 14 2015""` (Note the double double-quotes). CMake doesn't work with the double double-quotes.

I just wanted to confirm that this was an intentional change because all recipes adding single double quotes would now have double double-quotes and would break.